.ci/dockerfiles/Dockerfile.ubuntu22.04

```diff
@@ -0,0 +1,8 @@
+FROM harbor.mellanox.com/swx-infra/media/ubuntu_22.04:mofed-5.7-0.1.9.0 AS style
```

Comment:
why we need ofed image?

Reply:
for DPCP, which compilation is triggered in globals.sh
maybe for static tests, we need to change the approach and not compile anything